common/types/shared.go

```diff
@@ -9520,6 +9520,7 @@ type WorkflowExecutionInfo struct {
 	SearchAttributes  *SearchAttributes             `json:"searchAttributes,omitempty"`
 	AutoResetPoints   *ResetPoints                  `json:"autoResetPoints,omitempty"`
 	TaskList          string                        `json:"taskList,omitempty"`
+	IsCron            bool                          `json:"isCron,omitempty"`
```

Comment:
I think this type is also used by the response for DescribeWorkflowExecution API, so shall we also update that API handler to set this field?

Reply:
Thanks for that case. Let me take a look.